### ISSUE TITLE
Keyboard Layout: Proto-Indo-European

### ISSUE THREAD

**Issue by leszekjocz** [lingdb/ielex2#9]
As I understand, *.zip files are not accepted by Github, so I sent the layout to Paul and Cormac and asked them to forward is to you.

Description
*Installation:
All you have to do is to:
1) Unzip the folder;
2) Run "Setup.exe";
3) Wait a longer while (it often seems that the program is not running but in fact it is, so do not click on "Setup.exe" 100 times:);
4) Then, depending on the Windows version, you may have just to reboot your computer and/or add a layout manually (if so, look for "American international - custom" or "ie_ielex").
*Description:
The layout is built on the base of the "American international" layout.
I put all the diacritics/letters with diacritics to combination with the right alt key. For many of you it may by not usual but it has one great advantage: you have all "normal" letter, numbers and symbols on their "normal" place, so you don't have to switch between layouts when you write comments in English.
Where you find the symbols:
ʷ > R alt + w
ʰ > R alt + h
₁, ₂, ₃ > R alt + number 1, 2 or 3
̑ > R alt + number 6
̯ > R alt + number 8
̥ > R alt + number 0
You can also enter letters with the appropriate diacritic in one turn: r̥ > R alt + r, u̯ > R alt + u, g̑ > R alt + g, etc.

PS. I compiled the layout on Windows 8 and I knot that sometimes there are strange incompabilities in case of layouts. So if you have an older Windows version and this layout doesn't work for you, please let me know, I will compile another version on Windows 7.

**Comment by leszekjocz** [comment]
I changed the format of the *.zip file to *.pdf. Now I can attache it. Those who know how to do it, just change the format and double-click on it as usually. Alternatively you can run your compression tool, find the file and unzip it this way.
[ie_01.pdf](https://github.com/lingdb/ielex2/files/47328/ie_01.pdf)

**Comment by leszekjocz** [comment]
My email to Paul and Cormack was blocked because Gmail do not accept *.exe files in attachments (also when there are zipped)...
You can download the layout from my dropbox:
https://www.dropbox.com/s/uoc7c86lwrv7odj/ie_01.zip?dl=0

**Comment by m-scarborough** [comment]
Thanks Leszek!  For the Mac users, I still have my own keyboard layout which is available upon request.  (So far no one has asked but the offer remains open.)  NB: I've updated the PIE transcription policy on the wiki view of the presentation and discussion last Sunday.

**Comment by leszekjocz** [comment]
Ups, I forgot to add the acute. Update soon.

**Comment by leszekjocz** [comment]
New version (yesterday I forgot the accute), link above updated.

**Comment by leszekjocz** [comment]
Is "schwa" needful too?

**Comment by m-scarborough** [comment]
Not necessarily, but occasionally it's useful to have an extra schwa key.  (I've included one on my keyboard layout.)
